.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] [OPTIONAL] I suggest that you also add the `.vscode` directory to the `.gitignore` file. Since the project development environment settings should not be part of the project repo.